docs/UserGuide.md

```diff
@@ -69,123 +75,145 @@ Format: `help`
 
 Examples:
-*  `edit 1 p/81234567 a/Fifth Avenue`
-Edits the phone number and home address of the 1st person to be `81234567` and `Fifth Avenue` respectively.
-*  `edit 2 e/` 
-Edits the email address of the 2nd contact to be empty, thus clearing the email address field.
+*  `edit 1 p/81234567 e/jeremysand@glutter.com`
+edits the phone number and home address of the 1st person to be `81234567` and `jeremysand@glutter.com` respectively.
+*  `edit 2 n/Jayden Smith` 
+edits the name of the 2nd contact to be `Jayden Smith`.
 
-### Deleting a contact: `delete`
+#### Listing all contacts: `list`
 
-Deletes the specified developer or stakeholder from the contact list.
+Shows a list of all contacts in the contact list.
 
-Format: `delete INDEX`
+Format: `list`
 
-* Deletes the contact at the specified `INDEX`.
-* `INDEX` is the index number of the contact in the displayed contact list.
-* `INDEX` **must be a positive integer** 1, 2, 3, ...
+#### Finding contacts: `find`
+
+Finds the contacts whose names or descriptions contain any of the given keywords.
+
+Format: `find KEYWORDS`
+* `KEYWORDS` contains one or more keywords used to match contacts.
+* Searches only consider name and description.
+* Searches are case-insensitive, e.g. `hans` will match `Hans`.
+* Order of keywords does not matter, e.g. `Hans Bo` will match `Bo Hans`.
+* Searches only account for full words, e.g. `Han` will not match `Hans`.
+* Searches return contacts matching at least one keyword, e.g. `Hans Bo` will return `Hans Seed` and `Bo Yarns`.
 
 Examples:
+* `find John Kite` returns a contact with name `john kite` and another with description `Kite manufacturer`.
+* `find Sun` returns a contact with name `Joe Sun` and another with description `Works at Sun Tools`.
+
+#### Deleting a contact: `delete`
+
+Deletes the specified contact from the contact list.
+
+Format: `delete INDEX`
+* `INDEX` is the index number of the contact in the displayed contact list.
+* `INDEX` must be a positive integer.
+
+Example:
 * `delete 3` deletes the 3rd contact in the contact list.
 
+#### Clearing all contacts: `clear`
+
+Clears all contacts from the contact list.
+
+Format: `clear`
+
 ### Deliverable
 
-#### Adding a deliverable `add`
+#### Adding a deliverable: `add`
 
 Adds a deliverable to the deliverable list.
 
-Format: `add t/TITLE [desc/DESCRIPTION] [dead/DEADLINE] [c/CONTACT]…`​`
+Format: `add t/TITLE m/MILESTONE [d/DESCRIPTION] [by/DEADLINE] [c/CONTACTS]`
 * `TITLE` is the main heading of the deliverable.
-* `DESCRIPTION` is the additional details of the deliverable (e.g. sub-requirements). 
+* `MILESTONE` is the milestone tagged to the deliverable.
+* `MILESTONE` takes in numerical values separated by periods, e.g. `1.3`, `14.2.1`.
+* `DESCRIPTION` contains additional information about the deliverable, e.g. sub-requirements. 
 * `DEADLINE` is the due date time of the deliverable in DD-MM-YYYY HH:MM format.
```

Comment:
Consider using lower caps 'm' to differentiate between minutes and months (e.g `DD-MM-YYYY HH:mm`)